.travis.yml

```diff
@@ -202,8 +202,8 @@ matrix:
     - os: linux
       env:
         # These components will build as Habitat packages in the provided order
-        - COMPONENTS="hab-butterfly hab plan-build backline bintray-publish studio pkg-aci pkg-export-docker pkg-mesosize pkg-tarize sup"
-        - AFFECTED_DIRS="support/ci/deploy\.sh|Cargo\.toml|Cargo\.lock|VERSION|$_RUST_HAB_BIN_COMPONENTS|$_RUST_HAB_LIB_COMPONENTS"
+        - COMPONENTS="hab-butterfly hab plan-build backline bintray-publish studio pkg-aci pkg-dockerize pkg-cfize pkg-export-docker pkg-mesosize pkg-tarize sup"
```

Comment:
Could you also update `.bldr.toml` with any directory triggers for this component?

Reply:
Yep, I'll add that in here